demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+Running the demo with smaller models like `tiny-llama-1b-chat` requires approximately 4GB of available RAM.
+
+## Build image TODO
+
+Building the image with all required python dependencies is required. Follow the commands to build RedHat image:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make python_image BASE_OS=redhat OVMS_CPP_DOCKER_IMAGE=registry.connect.redhat.com/intel/openvino-model-server OVMS_CPP_IMAGE_TAG=2023.3
+```
+It will create an image called `registry.connect.redhat.com/intel/openvino-model-server:py`
+
+You can also build Ubuntu 22.04 image:
+```
+make python_image BASE_OS=ubuntu OVMS_CPP_DOCKER_IMAGE=openvino/model_server OVMS_CPP_IMAGE_TAG=2023.3
+```
+It will create an image called `openvino/model_server:py`
+
+
+## Download LLM model
+
+Download the model using `download_model.py` script:
+
+```bash
+cd demos/python_demos/rag_chatbot
+pip install -r requirements.txt
+
+python download_model.py --help
+INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
+usage: download_model.py [-h] --model
+                         {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+
+Script to download LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+                        Select the LLM model out of supported list
+
+python download_model.py --model ${SELECTED_MODEL}
+
+```
+The model will appear in `./tiny-llama-1b-chat` directory.
+
+## Weight Compression - optional
+
+[Weight Compression](https://docs.openvino.ai/canonical/weight_compression.html) may be applied on the original model. Applying 8-bit or 4-bit weight compression reduces the model size and memory requirements while speeding up execution by running calculations on lower precision layers.
+
+```bash
+INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
+usage: compress_model.py [-h] --model
+                         {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+
+Script to compress LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
```

Comment:
Do we need this parameter? Before we set `SELECTED_MODEL` env. I suppose the script could read that internally.